### ISSUE TITLE
Looking for a new package maintainer

### ISSUE THREAD

**Issue by wkranec** [wkranec/finance#34]
I'm looking for someone interested in taking over as the maintainer of the finance package. Send me a message on GitHub (or respond to this issue) if you are interested.

As much as I would like to continue to develop and maintain this code, the fact of the matter is that I no longer have the time to do so. There are several pull requests and good suggestions that have been left by the wayside due to my inaction, and I don't think that is right.

I originally wrote this code for a personal project, and I've always been flattered that the Ruby community has found it to be useful. Thanks!

**Comment by recurrence** [comment]
+1 on electing tubedude, his XIRR library is excellent.

**Comment by marksweston** [comment]
It looks like this conversation petered out with no further action.  @wkranec, if you're still around, could we arrange a handover to @tubedude if he's still interested or if not, talk about what else to do?  We use this library in production in our app, and I'd certainly like to help make sure it stays maintained.

**Comment by recurrence** [comment]
Times like this where you wish their was a community override for derelict repos. :)    Sure you can fork but until your fork becomes the 'community choice', that's all it is, a random fork.

**Comment by tubedude** [comment]
@marksweston, as @recurrence said, maybe it just a better idea to fork it and maintain your own since you are using it in production. I can sure contribute. I've learn a bit on the IRR method (which was my focus) and can bring some ideas.

**Comment by wkranec** [comment]
I am around and can arrange a handover.  Who is interested in taking over?

Thanks,
Bill
On Sep 2, 2015 12:24 PM, "Mark Weston" <notifications@github.com> wrote:

It looks like this conversation petered out with no further action. @wkranec
<https://github.com/wkranec>, if you're still around, could we arrange a
handover to @tubedude <https://github.com/tubedude> if he's still
interested or if not, talk about what else to do? We use this library in
production in our app, and I'd certainly like to help make sure it stays
maintained.

—
Reply to this email directly or view it on GitHub
<https://github.com/wkranec/finance/issues/34#issuecomment-137154267>.

**Comment by marksweston** [comment]
@wkranec, @tubedude thanks for the responses.

From a personal point of view I'm keen to see the gem maintained as it's important to our app and is pretty unique in the Ruby ecosystem.  I'd be happy to put in some time helping to make that happen, and taking on the role of maintainer if that was needed.  But I'm no more than an interested layman when it comes to finance, so I'd need some help both in terms of how the various calculations work and getting a better understanding of the design of the gem and the API.  So I would definitely need to take @tubedude up on your offer, and would be very grateful for any consultation @wkranec could offer during a handover.

I'll be talking to my employer in the next couple of days to make sure they're happy with me taking this on, and how much they're willing to help out with it.  Watch this space.

**Comment by marksweston** [comment]
OK, so if everyone's happy with this, I think it makes sense for my employer's github org (here: https://github.com/bcsgsvn) to take on this repository.  It's their app which uses this gem, and obviously it's in their interest to make sure it stays maintained.  Right now I'm the lead dev on that app and I'll expect to take the lead in getting things running here, but if and when I move on BCSG will still have the need and the dev resources to continue maintaining it.  

@wkranec if that makes sense to you, then I think we're ready for a hand-over.  There is one remaining question over licensing.  Previously, based on quickly skimming, I'd understood this gem to be licensed under LGPL v3 (and from your answer to issue #22 , I think that's your understanding too).  But the Copyright section at the bottom of the readme is actually rather ambiguous, mentioning LGPL, saying "all rights reserved", and then mentioning GPL v3 in three separate paragraphs.  Would you be able to make One Last Commit to clarify the copyright section before handing over?  That kind of change would be much better coming from the main copyright holder rather than being made *post hoc* by a new maintainer.

**Comment by marksweston** [comment]
@wkranec how about it?

**Comment by marksweston** [comment]
@wkranec ping!

**Comment by wkranec** [comment]
Mark,

I have committed an update to the README.  My original text was very much a
"cut and paste" from the GNU website, and didn't account for the intention
for the project to be licensed under the LGPL.  Please review and let me
know if any additional changes are required.

Thanks,
Bill

**Comment by marksweston** [comment]
Thanks Bill, that looks great.

Are you ready to transfer ownership of the repo?  I've just been looking through the docs here: https://help.github.com/articles/transferring-a-repository/

Transferring from a user to an organisation looks slightly complicated, so I think the best bet is to transfer the repo to my personal github account (marksweston) and then I can deal with any complications involved in moving it to my employer's organisation.

Mark

**Comment by wkranec** [comment]
Hi Mark,

I believe that I have successfully transferred the repository to your
personal account.  Please let me know if you do not see it.

Thanks,
Bill

**Comment by marksweston** [comment]
Yup, I've just accepted it.  Anyone watching will see it transfer again to my employer's github organisation  in the next day or two, as outlined above.

Thanks Bill.

**Comment by tubedude** [comment]
:+1:

**Comment by marksweston** [comment]
@wkranec 
Hey Bill + audience.  It turns out I missed something obvious; we'll need to be able to release new versions of the gem, but don't have any ownership rights on Rubygems.

http://guides.rubygems.org/command-reference/#gem-owner

I believe the correct command would be `gem owner finance mark.weston@bcsg.com`

This really should be the last thing you need to do.  I think...
